docs/topics/quickstart.md

```diff
@@ -1,5 +1,94 @@
-# Quickstart
+# Getting Started
+
+## Quickstart
 
 Currently, there is a [demo](https://github.com/SovereignCloudStack/cluster-stacks-demo) that can be used to see how the Cluster Stack approach can work. It uses the Docker Provider Integration for Cluster API.
 
-For production use, the Cluster Stack Operator can be used together with the [Cluster Stack Provider OpenStack](https://github.com/SovereignCloudStack/cluster-stack-provider-openstack) to manage Kubernetes clusters on OpenStack. You can see how it is used in the corresponding [Quickstart Documentation](../../cluster-stacks/providers/openstack/quickstart.md) of Cluster Stacks.
+## Installation
+
+### Prerequisites
+
+```
+- clusterctl
+- envsubst
+- kubectl
+- Helm (optional)
+```
+
+#### Cluster API
+
+We assume access to a Kubernetes cluster.
+This Kubernetes cluster must act as Cluster API management cluster.
```

Comment:
Should we add a hint that `kind` or `k0s` or `k3s` can be used to create a lightweight management cluster to bootstrap the setup (and to move the cluster-API objects to another cluster later)?